readthedocs/projects/views/private.py

```diff
@@ -113,6 +114,28 @@ def get_context_data(self, **kwargs):
             # Alternatively, dynamically override super()-derived `project_list` context_data
             # context[self.get_context_object_name(filter.qs)] = filter.qs
 
+            projects = AdminPermission.projects(user=self.request.user, admin=True)
+            if (timezone.now() - self.request.user.date_joined).days < 30:
+                context[
+                    "promotion"
+                ] = "includes/elements/promotions/example-projects.html"
+            elif (
+                projects.exists()
```

Comment:
It's probably worth consolidating check logic so we're not duplicating queries, this page already loads quite slow due to the admin permission checks.